hackathon_site/dashboard/frontend/src/pages/Teams/Teams.test.tsx

```diff
@@ -21,7 +21,7 @@ describe("Teams Page", () => {
         const teamsApiResponse = makeMockApiListResponse(mockTeams);
 
         when(mockedGet)
-            .calledWith(teamsUri, { limit: NUM_TEAM_LIMIT })
+            .calledWith(teamsUri, { limit: NUM_TEAM_LIMIT, search: "" })
```

Comment:
from the comments in the slice file, you probably don't need the search property here anymore